src/Schema/Context.php

```diff
@@ -12,15 +12,23 @@ class Context implements GraphQLContext
     /**
      * An instance of the currently authenticated user.
      */
-    public ?Authenticatable $user;
+    public ?Authenticatable $user = null;
```

Comment:
```suggestion
    public ?Authenticatable $user;
```

Reply:
If there is no guard at all, calling $context->user() would result in following error:
> Error : Typed property Nuwave\Lighthouse\Schema\Context::$user must not be accessed before initialization

I now added a $this->user = null; before the foreach loop.